<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1521265103175' stop='1521265110379'>
    <name>To test the Angular binding functionality</name>
    <title>To test the Angular binding functionality</title>
    <test-cases>
        <test-case start='1521265103176' status='passed' stop='1521265107102'>
            <name>should not have any previous results</name>
            <title>should not have any previous results</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='748ae500-4e4f-4de7-b5f2-cffbb6411eb2-attachment.png' type='image/png' size='15605'/>
            </attachments>
        </test-case>
        <test-case start='1521265107103' status='failed' stop='1521265110379'>
            <name>Should display calculation results for performed Arthematic operations and should save results in table</name>
            <title>Should display calculation results for performed Arthematic operations and should save results in table</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='cb39a11b-04c9-4514-a621-ebdd0b84df3f-attachment.png' type='image/png' size='18269'/>
            </attachments>
            <failure>
                <message>Expected '1' to equal '3'.</message>
                <stack-trace>Error: Failed expectation
                    at C:\Users\sandeep\Desktop\Protractor\Protractor\specs\calculation_spec.js:34:34
                    at ManagedPromise.invokeCallback_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at asyncRun (C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
                    at C:\Users\sandeep\Desktop\Protractor\Protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:118:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>